.infra/Pulumi.prod.yaml

```diff
@@ -95,7 +95,6 @@ config:
     personalizedDigestFeed: http://feed-digest-server.feed/api/personalised
     personalizedDigestSecret:
       secure: AAABAN1TYaqznVMVbKDZtYEnLGB0VIzXDrpcmB6rBsImneobrDOS12T4XwbVvnexdZt1NBCnZRBwH4AL7OgpZkj/2UgT8Lzj0o7TdVSBeIeVDzKa/V6jlvJ4AskEC9rg
-    popularFeed: http://feed.feed.svc.cluster.local/api/popular
```

Comment:
We're still using this endpoint though. Check the anonymous feed query, it uses the `feedGenerators['popular']` which refer to this endpoint

Reply:
I search and I don't see any reference to `env.POPULAR_FEED` in codebase aside from what I removed